### PR TITLE
Add .travis.yml to run tests

### PR DESCRIPTION
**Pull request by ThomasG77**
I added the `.travis.yml` file. It can run the tests.
The other purpose of this PR in the long run is also to add plugins installation tests as they fails depending of environment.

I should mention I already tested the configuration at https://travis-ci.org/ThomasG77/kosmtik where it fails due to some kosmtik tests not working ;)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+  - "5.5"
+  - "4.2"
+  - "0.12"
+  - "0.10"
+git:
+  depth: 10
```
Comment (yohanboniface) [inline .travis.yml]: Why that depth specifically? :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+before_install:
+    - sudo add-apt-repository -y ppa:mapnik/nightly-trunk
```
Comment (yohanboniface) [inline .travis.yml]: Isn't mapnik supposed to be installed embedded by the node js package?

**Comment by yohanboniface** [comment]
Good point about installing and testing the plugins a bare minimum! I'd love to go this way.

**Comment by ThomasG77** [comment]
You are right for the second point. I removed it.
For the first point, it's arbitrary set to 10. It's to improve project download time.
The default git depth is 50 on Travis e.g https://docs.travis-ci.com/user/customizing-the-build/#Git-Clone-Depth

I can submit a new PR with only one commit when everything is OK for you.

**Comment by yohanboniface** [comment]
Sounds good, let's iterate on this :)

**Comment by ThomasG77** [comment]
Did you activate the build from kosmtik/kosmtik repository? Because https://travis-ci.org/kosmtik/kosmtik/ empty

Do you also need some help for automating kosmtik npm publication from Github e.g http://mclear.co.uk/2013/10/06/publishing-to-npm-on-git-commit-using-github-hooks-and-travis/ ? The current version is 0.0.5 on https://www.npmjs.com/package/kosmtik-mbtiles-export whereas it's 0.0.6 on Github. It seems you do it manually ;)

**Comment by yohanboniface** [comment]
> Did you activate the build from kosmtik/kosmtik repository? Because https://travis-ci.org/kosmtik/kosmtik/ empty

Not yet, I'm at work ;)

> Do you also need some help for automating kosmtik npm publication from Github

Could be nice, yes, thanks :)

**Comment by ThomasG77** [comment]
As I don't have access to your `.npmrc` key, I prefer to provide a script based on the article in my previous command at https://gist.github.com/ThomasG77/57b8fea87d874ec1c2c5
To try, you may start with one repository and/or remove the last line to avoid commit from git.

You also need to choose the deploy strategy e.g when you tag or when you make a commit on master https://docs.travis-ci.com/user/deployment/npm#What-to-release

**Pull request merged**